website/docs/getting-started/installation-and-setup.md

```diff
@@ -74,7 +75,7 @@ module.exports = {
 ```
 
 This configuration also can be specified in `"relay"` section of the `package.json` file.
-For more details, and configuration options see: [Relay Compiler Configuration](https://github.com/facebook/relay/tree/main/packages/relay-compiler)
+For more details, and configuration options see: [Relay Compiler Configuration](https://github.com/facebook/relay/tree/main/packages/relay-compiler/index.d.ts)
```

Comment:
Love this! Could you also add a similar note to https://github.com/facebook/relay/blob/main/packages/relay-compiler/README.md#configuration?

Reply:
Added! I updated the `README` with a note admonition containing the same info.